parts/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -217,8 +217,10 @@ spec:
       hostNetwork: true
       serviceAccountName: canal
       tolerations:
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
+        - key: "node-role.kubernetes.io/master"
```

Comment:
I think so. 

Just want to keep it consistent. I noticed in this file. there is another toleration.
```
         - key: "CriticalAddonsOnly"
            operator: "Exists"
```

They are quoted.